### ISSUE TITLE
Update idea hub related feature tours to have correct versions

### ISSUE THREAD

**Issue by eugene-manuilov** [google/site-kit-wp#4066]
## Feature Description

We have two feature tours for Idea Hub module. We need to update their versions to make sure it equals the plugin version that releases the Idea Hub functionality.

https://github.com/google/site-kit-wp/blob/a4a6476e911c83d662faa1d72b0423fb0d4dbb25/assets/js/feature-tours/idea-hub-module.js#L42-L43

https://github.com/google/site-kit-wp/blob/a4a6476e911c83d662faa1d72b0423fb0d4dbb25/assets/js/feature-tours/idea-hub-posts-table.js#L34-L35

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Both Idea Hub feature tours should only trigger if the user's "initial version" is below `1.43.0`.
    * In other words, if someone starts using Site Kit at 1.42.0, they should still see the feature tours, as it will only be rolled out after that launch.

## Implementation Brief

* Update the versions of both Idea Hub feature tours to be `1.43.0`

### Test Coverage

* No changes

## QA Brief

* Ensure the version in both instances is updated to `1.43.0`, as specified, in the `main` branch.

## Changelog entry

* N/A

**Comment by aaemnnosttv** [comment]
@felixarntz should these be 1.42.0 or 1.43.0?

**Comment by felixarntz** [comment]
@aaemnnosttv 1.43.0 makes most sense to me, I've added ACs.

**Comment by tofumatt** [comment]
IB ✅

**Comment by eugene-manuilov** [comment]
QA :heavy_check_mark: both versions have been updated:

https://github.com/google/site-kit-wp/blob/37e930fe4e8d97a8e1eea7237c4d74c9c622e74e/assets/js/feature-tours/idea-hub-module.js#L42
https://github.com/google/site-kit-wp/blob/37e930fe4e8d97a8e1eea7237c4d74c9c622e74e/assets/js/feature-tours/idea-hub-posts-table.js#L34
